install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -34,7 +34,8 @@
 
 {{- /* Default values when 1.10 was initially deployed */ -}}
 {{- if semverCompare ">=1.10" (default "1.10" .Values.upgradeCompatibility) -}}
-{{- $defaultKubeProxyReplacement = "disabled" -}}
+  {{- $defaultKubeProxyReplacement = "disabled" -}}
+  {{- $defaultBpfMasquerade = "false" -}}
```

Comment:
please add a comment here stating that this needs to be done because `semverCompare >= 1.8` is also hit and `$defaultBpfMasquerade` is set as true on line 22